request_a_govuk_domain/request/admin/model_admins.py

```diff
@@ -158,8 +158,6 @@ def export(self, _request, queryset):
                     row.append(self.format_date(obj.time_submitted))
                 elif field == "application_month":
                     row.append(self.get_application_month(obj.time_submitted))
-                elif field == "org_type":
```

Comment:
Why was this removed? If it's unrelated to add_time_decided_local_time please make a separate commit.

Also can you make both commit messages clearer as to the reason why those changes were made? And also, a reminder about the 50/72 rule.